websites/R/R-a-d.io/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "191388216509595648",
+		"name": "awooooool"
+	},
+	"service": "R-a-d.io",
+	"description": {
+		"en": "An online radio focused on Japanese songs"
```

Comment:
This doesn't seem like the **official** description

Reply:
It is indeed, I can't seem to find one. Should I contact them or something?

Reply:
How about the description? I made that myself because they don't have their "About Us" page on their website. Should I just leave it like that or do I need to contact them or something?

Reply:
Ah, I overlooked the fact that they have Android app. Thank you so much for pointing out.